README.md

```diff
@@ -20,25 +19,26 @@ Firefly APIs exposes the most powerful components of Firefly in a way that can b
 An important feature is that the Firefly APIs server can be installed cross-site. In other words, it is not required to be on the same server as the web page. Firefly APIs can do this because it uses JSONP or CORS for the server communication. This allows Firefly APIs not to be limited by the server's Same Origin Policy and to give the developer a lot of flexibility. The web developer does not have to do any installation, but can simply just start using Firefly APIs.
 
 ### Visualizers
-The firefly components contain 3 main visualizers for astronomy data.
+The firefly components contain three main visualizers for astronomy data.
 
 
-#### Tabular Display
-Firefly has implemented “Excel-like” tables on the webpage. In an easy, interactive way, a user can sort the results, filter the data on multiple constraints, hide or show columns or select data for download or visualization. The Firefly APIs server is optimized to show very large tables without significant performance degradation. We have tested it with over 10 million rows, 50 column tables.
+#### Tabular Data Display
+Firefly has implemented “Excel-like” tables on the webpage. In an easy, interactive way, a user can sort the results, filter the data on multiple constraints, hide or show columns, or select data for download or visualization. The Firefly APIs server is optimized to show very large tables without significant performance degradation. We have tested it with over 10 million rows, 50 column tables.
 
 #### FITS Visualization
-Firefly provides a first-class FITS visualization on the Web without any plugins. All of the significant components you would expect to see in a basic desktop FITS application are available with data that Firefly displays. The FITS visualizer allows for plotting anything that the table shows that has a Lon and Lat. Therefore, users can overlay multiple catalogs over their FITS image and interact with it.
+Firefly provides a first-class FITS visualization on the Web without any plugins. All of the significant components you would expect to see in a basic desktop FITS application are available with data that Firefly displays. The FITS visualizer allows for plotting any table data with Lon(RA) and Lat(Dec). Therefore, users can overlay multiple catalogs over their FITS image and interact with it.
+
+#### 2D Graphs
+Firefly shows 2D graphs interactively so that a user can read the data point values as he moves his mouse around or zooms in to investigate the data at a finer level. These graphs are used for spectrum or plotting table columns. The user can specify any column that the tables are showing. The user can also define expressions from the columns for the XY Plot, such as (col1*col2/tan(col3)). Firefly also supports histogram and heatmap graph. 
 
```

Comment:
This is just my opinion. I would be more generic about charts, since we support them all now.

```
Graphs

Firefly can display any Plotly chart. The data can be static or derived from a table. When defining chart data, users can refer to table columns or table column expressions, such as (col1*col2/tan(col3)). Table based charts share their data model with the table and can trigger or respond to the common events, such as filtering of the data.
```

Reply:
Fixed typo and changes graph section. But I will leave the "support all ployly charts to next update since we don't have all the UI ready yet.